projects/packages/forms/src/dashboard/app.js

```diff
@@ -0,0 +1,13 @@
+import { Provider } from 'react-redux';
```

Comment:
Should we move this into `components/app` ?

Reply:
As suggestions go this one is quite lame :D do you want it moved to components/app?